File: gen_scripts/perl/e00.branch.pl
#!/usr/bin/env perl
# get input from user
print "Would you like a toast? [Yes/No]: ";
my $response=<>;               # acquire string input including newline
chomp $response;               # strip newline

# set response string using if/else construction
#   Test response to a string
if ($response eq "Yes") {
  $response_str = "That's great!\n";
} else {
  $response_str = "How about a muffin?\n";
}
# output the response string
print $response_str;

File: gen_scripts/python2/e00.branch.py
#!/usr/bin/env python
# get input from user
response = raw_input("Would you like a toast? [Yes/No]: ")

# set response string using if/else construction
#   Test response to a string
if response == "Yes":
  response_str = "That's great!"
else:
  response_str = "How about a muffin?"

# output the response string
print response_str